kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/VirtualKafkaClusterReconciler.java

```diff
@@ -61,13 +69,16 @@ public final class VirtualKafkaClusterReconciler implements
         Reconciler<VirtualKafkaCluster> {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(VirtualKafkaClusterReconciler.class);
-    public static final String PROXY_EVENT_SOURCE_NAME = "proxy";
-    public static final String PROXY_CONFIG_STATE_SOURCE_NAME = "proxy-config-state";
-    public static final String SERVICES_EVENT_SOURCE_NAME = "services";
-    public static final String INGRESSES_EVENT_SOURCE_NAME = "ingresses";
-    public static final String FILTERS_EVENT_SOURCE_NAME = "filters";
-    public static final String TRANSITIVELY_REFERENCED_RESOURCES_NOT_FOUND = "TransitivelyReferencedResourcesNotFound";
-    public static final String REFERENCED_RESOURCES_NOT_FOUND = "ReferencedResourcesNotFound";
+    static final String PROXY_EVENT_SOURCE_NAME = "proxy";
+    static final String PROXY_CONFIG_STATE_SOURCE_NAME = "proxy-config-state";
+    static final String SERVICES_EVENT_SOURCE_NAME = "services";
+    static final String INGRESSES_EVENT_SOURCE_NAME = "ingresses";
+    static final String FILTERS_EVENT_SOURCE_NAME = "filters";
+    static final String KUBERNETES_SERVICES_EVENT_SOURCE_NAME = "kubernetesServices";
+    static final String TRANSITIVELY_REFERENCED_RESOURCES_NOT_FOUND = "TransitivelyReferencedResourcesNotFound";
+    static final String REFERENCED_RESOURCES_NOT_FOUND = "ReferencedResourcesNotFound";
+    public static final String VIRTUALCLUSTER_NAME_ANNOTATION = "io.kroxylicious.operator.ingress.virtualcluster-name";
+    public static final String INGRESS_NAME_ANNOTATION = "io.kroxylicious.operator.ingress.ingressr-name";
```

Comment:
* Can we use the usual naming scheme? I.e. I think this should be `io.kroxylicious/operator.ingress.virtualcluster-name`, etc.
* I think we should be documenting _somewhere_ what these things are/how they're used.